chord_metadata_service/chord/migrations/0005_projectjsonschema_and_more.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 4.2 on 2023-05-01 14:06
```

Comment:
rename this migration to v3_0_0 since we should squash all relevant migrations into a single migration per app for each version